sdk/src/validation_results.rs

```diff
@@ -21,20 +23,40 @@ use crate::{
     jumbf::labels::manifest_label_from_uri,
     status_tracker::{LogKind, StatusTracker},
     store::Store,
-    validation_status::{log_kind, ValidationStatus},
+    validation_status::{self, log_kind, ValidationStatus},
 };
 
 #[derive(Copy, Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
 #[cfg_attr(feature = "json_schema", derive(JsonSchema))]
-/// Indicates if the manifest store is valid and trusted.
+/// Represents the levels of assurance a manifest store achives when evaluated against the C2PA
+/// specifications structural, cryptographic, and trust requirements.
+///
+/// See [§14.3. Validation states].
 ///
-/// The Trusted state implies the manifest store is valid and the active signature is trusted.
+/// [§14.3. Validation states]: https://spec.c2pa.org/specifications/specifications/2.2/specs/C2PA_Specification.html#_validation_states
 pub enum ValidationState {
```

Comment:
We need to add a NotValidated state since our SDK has an option to not validate. Currently if someone enables that setting, we end up labeling the manifest store as Trusted, but we can't state anything about it other than NotValidated if we don't validate.